config.js

```diff
@@ -64,9 +64,19 @@ module.exports = {
 
     // Specifies the behavior of the trace agent in the case of an uncaught exception.
     // Possible values are:
-    //   `ignore`: Take no action.
-    //   `flush`: Attempt to publish traces silence the exception.
-    //   `flushAndExit`: Attempt to publish traces and exit the process.
+    //   `ignore`: Take no action. Note that the process may termiante before all the 
```

Comment:
README should be update too, but it is important for the config to be descriptive too, specially that we point people to the config.js file for documentation on the configuration options.